src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -6,7 +6,7 @@
            [java.nio.file.attribute FileAttribute PosixFilePermissions]
            (java.security PrivateKey PublicKey KeyPair)
            (org.joda.time DateTime)
-           (java.security.cert CRLException)
+           (java.security.cert CRLException X509CRL)
```

Comment:
Is this actually getting used?

Reply:
Yeah, it's where `.getNextUpdate` comes from.

Reply:
ahh 👍